tasks/release.rb

```diff
@@ -61,6 +68,7 @@
 
     task :push => :build do
       sh "gem push #{gem}"
+      sh "npm publish" if File.exist?("#{framework}/package.json")
```

Comment:
I pinged @jeremy on Basecamp to ask if he could help me with getting Core setup with npm accounts, and I never heard back 😬